doc/samples/SqlConfigurableRetryLogic_StepByStep_CustomProvider.cs

```diff
@@ -0,0 +1,255 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Data.SqlClient;
+
+namespace CustomCRL_Doc
+{
+    class Program
+    {
+        private const string CnnStringFormat = "Server=localhost; Initial Catalog=Northwind__; Integrated Security=true; pooling=false; Timeout=1";
```

Comment:
Is the "__" prefix intentional?

Reply:
It's not included in Snippet blocks. But, I'd rather to remove it. 👍